### PR TITLE
Plant branch

### PR DESCRIPTION
**Pull request by SNora00**
Hi!
Made the Plants class and another 3 classes which extend from Plant class, could you guys review my code?
Thanks

### CODE REVIEW

File: stellar_dew.iml [hunk 1]
```diff
@@ -4,25 +4,8 @@
     <exclude-output />
     <content url="file://$MODULE_DIR$">
       <sourceFolder url="file://$MODULE_DIR$/src" isTestSource="false" />
-      <sourceFolder url="file://$MODULE_DIR$/test" isTestSource="true" />
```
Comment (pfazekas1) [inline stellar_dew.iml:7]: Why is this stuff deleted? Is the folder for the test not right here?

**Review by pfazekas1 — COMMENTED** [review]
Hello. The code seems to be good but why is the menu commited to a branch called Plant?

**Comment by SNora00** [comment]
My branch name was proba, and I wanted to make it more representative, should I name it back?

File: stellar_dew.iml (hunk 1 above)
Comment (SNora00) [reply stellar_dew.iml:7]: I don't know, I didn't deleted it manually.

**Comment by pfazekas1** [comment]
On further review and after consulting with the teachers we should not merge this branch into the main branch. You should make a new branch and use "git cherry-pick" to put only the plant related commits on the new branch, then make a new pull request for that. This way we can avoid problems.

**Comment by SNora00** [comment]
I had some conflicts in the code and recently I'm trying to solve them... 
Thank you for your feedback

**Pull request closed without merging**